<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QMainWindow" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Youtube URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_edit_youtube_url"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ok">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textedit_user_input"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>Status : Null</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>InputWindow</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_edit_youtube_url</sender>
   <signal>returnPressed()</signal>
   <receiver>InputWindow</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>download()</slot>
 </slots>
</ui>
